includes/fbproduct.php

```diff
@@ -448,15 +472,78 @@ public function get_fb_description() {
 		return apply_filters( 'facebook_for_woocommerce_fb_product_description', $description, $this->id );
 	}
 
+	/**
+	 * Get the rich text description for a product.
+	 *
+	 * This function retrieves the rich text product description based on the following logic:
+	 * 1. Check if the facebook rich text description is set and not empty.
+	 * 2. If the rich text description is available, use it as the preferred description.
+	 * 3. Otherwise, fall back to the plain text description made available by Woocommerce.
```

Comment:
```suggestion
	 * 1. Check if the Facebook rich text description is set and not empty.
	 * 2. If the rich text description is available, use it as the preferred description.
	 * 3. Otherwise, fall back to the plain text description made available by WooCommerce.
```